app/script/conversation/ConversationRepository.coffee

```diff
@@ -112,12 +112,13 @@ class z.conversation.ConversationRepository
   @param conversation_id [String] Conversation ID
   @param generic_message [z.protobuf.GenericMessage] Protobuf message to be encrypted and send
   @param user_ids [Array<String>] Optional array of user IDs to limit sending to
+  @param native_push [Boolean] Optional if message should enforce native push
   @return [Promise] Promise that resolves when the message was sent
   ###
-  _add_to_sending_queue: (conversation_id, generic_message, user_ids) =>
+  _add_to_sending_queue: (conversation_id, generic_message, user_ids, native_push) =>
```

Comment:
Do you think it would be good to default this one to `false` to avoid conflicts in the future?